### ISSUE TITLE
Loopback middleware final phase broken by setup

### ISSUE THREAD

**Issue by Ozitiho** [BoLaMN/loopback-datatype-objectid#2]
We're using custom error catching in our project. To do this we call a simple piece of code in the loopback middleware in the `final:after` step. However, using this module makes the entire `final` middleware step un-editable. For example, we have the following setting:

```
  "final": {
    "loopback#urlNotFound": {}
  },
  "final:after": {
    "./error-logger": {},
    "strong-error-handler": {
      "params": {
        "log": false,
        "debug": true
      }
    }
  }
```

But `error-logger` isn't called, and the params for `strong-error-handler` are defaulted to `log: true` and `debug: false`. I confirmed this issue in a clean loopback 3.x project. 

Reverting to 1.0.0 appears to resolve this issue.

**Comment by BoLaMN** [comment]
Are you able to share that clean loopback project and the scenario as where it fails? 

The only versions I use are highly modified and can’t seem too replicate

**Comment by Ozitiho** [comment]
Sure can, I made a public git displaying the issue.

https://github.com/Ozitiho/loopback-objectid-test

I added a README with further description and steps to replicate. Hope it's clear enough.

**Comment by BoLaMN** [comment]
just published 1.0.5 which fixes the issue https://github.com/BoLaMN/loopback-datatype-objectid/commit/e2f7ec32af5758fc84d3b2d32cbbed3bf96b29a0
